<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WineDriveDialog</class>
 <widget class="QDialog" name="WineDriveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wine Drive settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Wine Drive</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>-1</number>
      </property>
      <property name="margin">
       <number>3</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_14" native="true">
        <layout class="QHBoxLayout" name="_5">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Drive letter:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbDriveLetter">
           <item>
            <property name="text">
             <string notr="true">A:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">B:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">C:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">D:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">E:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">F:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">G:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">H:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">I:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">J:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">K:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">L:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">M:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">N:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">O:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">P:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Q:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">R:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">S:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">T:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">U:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">V:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">W:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">X:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Y:</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">Z:</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <layout class="QHBoxLayout" name="_3">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Path:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtDrivePath"/>
         </item>
         <item>
          <widget class="QPushButton" name="cmdGetDrivePath">
           <property name="text">
            <string/>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_13" native="true">
        <layout class="QHBoxLayout" name="_2">
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>140</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>140</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbDriveType">
           <item>
            <property name="text">
             <string notr="true">auto</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">hd</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">floppy</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">network</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">cdrom</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cmdCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cmdOk">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
